org.sithlord48.BlackChocobo.yml

```diff
@@ -0,0 +1,58 @@
+app-id: org.sithlord48.BlackChocobo
+default-branch: stable
+runtime: org.kde.Platform
+runtime: org.kde.Sdk.Locale
+sdk: org.kde.Sdk
+runtime-version: 6.4
+command: blackchocobo
+finish-args:
+  - --filesystem=host
```

Comment:
that's a very broad permission

Reply:
Is there a more limited way to allow r/w of user files

Reply:
Changed to home